build.rs

```diff
@@ -277,6 +277,8 @@ fn main() {
         }
     }
 
+    println!("cargo:rustc-env=SGX_ENCLAVE_DEVICE=/dev/sgx_enclave");
```

Comment:
What?
Why?
How can someone overwrite this?

Reply:
Why should someone overwrite this?

Reply:
Maybe what you wanted was a 
```rust
const SGX_ENCLAVE_DEVICE: &str = "/dev/sgx_enclave";
```
?

Reply:
So the point is to define the path in a build.rs. Are those constants also available for compilation units?

Reply:
I still think this should be removed, especially since we now have a constant defined